src/minio/managers/policy_manager.py

```diff
@@ -702,31 +699,37 @@ async def _load_minio_policy(self, policy_name: str) -> PolicyModel:
         # Create PolicyDocument from the policy JSON
         statements = []
         for stmt_data in policy_json.get("Statement", []):
-            actions = []
```

Comment:
What's the difference between this code and PolicyDocument.from_dict()? Can you just call that here?

Reply:
👍